cellmlmanip/model.py

```diff
@@ -153,7 +182,7 @@ def add_number(self, value, units):
         Creates and returns a :class:`NumberDummy` to represent a number with units in sympy expressions.
 
         :param number: A number (anything convertible to float).
-        :param units: A string unit name or a ``Unit`` object.
+        :param units: A string unit name or a :class:`~cellmlmanip.units.UnitStore.Unit` object.
```

Comment:
What's the ~ for?

Reply:
It means it shows just as `Unit` in the link text, rather than the full dotted name.